sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/source/CosmosChangeFeedModes.java

```diff
@@ -18,7 +18,7 @@ public String getName() {
 
     public static CosmosChangeFeedModes fromName(String name) {
         for (CosmosChangeFeedModes mode : CosmosChangeFeedModes.values()) {
-            if (mode.getName().equals(name)) {
+            if (mode.getName().equalsIgnoreCase(name)) {
```

Comment:
why - we always write it - why would we need to be case-insensitive?

Reply:
either "allVersionxxx" or "ALLVERSIONxxx" should all be considered valid, so changed to use ignoreCase here